src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,14 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 1 || value == 0) {
+            return "" + value;
+        }
+        StringBuilder resultStr = new StringBuilder();
+        for (int i = value; i > 0; i /= 2) {
+            resultStr.append(i % 2 == 0 ? 0 : 1);
```

Comment:
what result can the expression return: i % 2? 
Hint:
you shouldn't use ternary operator here

Reply:
of course, 1 or 0. 
fixed